tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile.py

```diff
@@ -239,8 +279,13 @@ def init_dependencies_for_scala_libraries(self):
       jars=[JarDependency(org='com.example', name='scala', rev='0.0.0')]
     )
 
-  def create_task_with_target_roots(self, target_roots):
-    context = self.context(target_roots=target_roots)
+  def create_task_with_target_roots(self, target_roots, default_workflow=None):
```

Comment:
I might remove the `default_workflow=None` if it's not currently being used.

Reply:
Ah. that reveals a test I should write!